src/temporal.jl

```diff
@@ -0,0 +1,23 @@
+"""
+    CustomTransform <: Transform
```

Comment:
is this a typo or are you planning on implementing it as part of HoD?

Reply:
typo